### PR TITLE
Added demo page

### PR DESCRIPTION
**Pull request by ghost**
@AdamJosephKalimi

### CODE REVIEW

**Comment by ghost** [comment]
@AdamJosephKalimi 
I put the components in the designated folder and also left them on the demo page so you could see what they look like.

**Comment by ghost** [comment]
I added flexbox to center the arrow icon to the text whereas before it was centered with eyeballed pixel values.

File: app.json [hunk 1]
```diff
@@ -9,6 +10,5 @@
     "navigationBarTitleText": "WeChat",
     "navigationBarTextStyle":"black"
   },
-  "style": "v2",
```
Comment (AdamJosephKalimi) [inline app.json:12]: @Frank-Xin Why was this deleted?

**Comment by AdamJosephKalimi** [comment]
@Frank-Xin 
Looks like good progress has been made on the buttons. 

I think Button Style **G** is missing. Can you please include that.

I also left a comment on line 12 of app.json. Please have a look at than and tag me in your response.

Good work already put all of the components into their own files under the component folder.

Once you address above, let me know and I will review and merge 🚀

**Comment by ghost** [comment]
@AdamJosephKalimi I was unable to comment out style: v2 as it's in a json document--I did some researching and found that the line helps define default styling for some WXML elements and was added to WeChat Devtools at the end of 2019.

**Comment by ghost** [comment]
@AdamJosephKalimi Sorry about not separating the commits into different pull requests--Github puts all my commits into the same request automatically and I wasn't sure how to separate them without creating new branches.

**Comment by AdamJosephKalimi** [comment]
@Frank-Xin 
Glad to see you were able to get through the key tasks for today.
I have reviewed the code and will list all of the feedback points here. Please implement them and make a new pull request.

**Card component**
_Good that we have the initial framework created. Adjustments need to be made to maintain the proportions designed in the Figma file._
        - Profile photo - general rule for px to rpx is that 1 px == 2 rpx. So if the image in Notion is 55px x 55px, the profile photo in the code should be 110rpx x 110rpx. It is currently 160rpx x 160rpx.
        - The font sizes need to be adjusted to reflect the font sizes in Notion. For example the font size for card-title is 50rpx, while the font in the Notion is 22px. The font size in the code should be 44px. 
        - Please go through the code for this component and make the appropriate updates so the dimensions are consistent with the design.
        - The component itself should have a longer width to height ratio, as shown in the Figma file. Please make the corresponding adjustments. Feel free to tag myself or Audrey in any roadblocks that you hit.

**Home-directory**
        - Right now your wxml is all left-aligned without indentation. Did you originally code it that way? It makes it very difficult for any person looking at your code to interpret what the hierarchy is. See the photo attached at the bottom. Please update with appropriate indentation.
        - One thing I asked you to avoid was not including the **current user’s name** at the top of this page - I’ve commented it out, but it caused the “Browse All” and “Filter by creative areas” buttons to shift. First, create a container that surrounds the two buttons so they act as a single element in the overall hierarchy of the wxml code. Then comment out the line for the dynamic field for the current user (currently line 4).


**Miscellaneous**
         - I am unable to download the fonts into the Mini Program. The server responded with a status of 403 (HTTP/1.1 403 Forbidden) - From server 52.218.252.152 - The request to load the fonts is being denied by the server. Strange bc it was working on Tuesday when I was checking the buttons created on Wednesday. Is this a problem for you as well? 

**Conclusion**
         - Overall, I am happy to see that you were able to get through a lot of these cards
         - You did well implementing the seed data and looping over it to render the content dynamically
         - The **components-button(s)** card and **Assets Folder** card have been moved over to Completed. Once you implement the changes above, I will move the other cards over to completed.
         - There is quite a lot of critical feedback above, but you should see this in a positive light. We want you to focus more on attention to detail. This is what separates developers who can get things working from developers who build great products. Specifically, I am saying that you should analyze the Figma file with more focus when you're implementing the design in code. Follow the proportions and it should look great upon the next review.

Screen Shot 2020-07-15 at 2 18 19 PM
